visivo/query/runner.py

```diff
@@ -82,10 +82,9 @@ def run(self):
     def _run_trace_query(self, queue: Queue):
```

Comment:
The other thing that this change unlocks is being able to parallelize query runs  across different targets in addition to threading them within the target now that the dag flows from target to model to trace.